src/localization/localization.ts

```diff
@@ -157,47 +157,47 @@ const localization = {
             monthNameYear: (date: Date) => `${getMonthName(date.getMonth())} ${date.getFullYear()}`
         },
         barrel: {
-            addAnother: "Add another project/styleguide",
-            selectType: "Please select if you would like to add project or styleguide",
+            addAnother: "Add another project or styleguide",
+            selectType: "Add project or styleguide?",
             addProject: "Add project",
             addStyleguide: "Add styleguide",
-            added: (type: BarrelType) => `Added ${barrelLower(type)} to sidebar.`
+            added: (type: BarrelType) => `Added ${barrelLower(type)}.`
         },
         jira: {
-            open: "Open JIRA",
-            select: "Select JIRA"
+            open: "Open Jira issue",
+            select: "Select Jira issue"
         },
         jumpTo: {
             jumpToItem: "Jump to screen or component",
             selectItem: "Select screen or component",
             noItemFound: "No screen or component found"
         },
         openInZeplin: {
-            selectPreferred: "Select preferred application type to open items in Zeplin",
-            web: "Open in Web App",
-            app: "Open in Native App"
+            selectPreferred: "Select where to open Zeplin resources",
+            web: "Open in Web",
+            app: "Open in App"
         },
         pin: {
             selectProject: "Select project",
-            noProjectFound: "Please add a project to sidebar first.",
+            noProjectFound: "Please add a project or styleguide first.",
```

Comment:
This is used on pinning screens. Screens are only found in projects, so "or styleguide" part should be removed